library/ssl_tls13_generic.c

```diff
@@ -4870,6 +4884,7 @@ void mbedtls_ssl_del_client_ticket( mbedtls_ssl_ticket *ticket )
     mbedtls_platform_zeroize( ticket->key, sizeof( ticket->key ) );
 }
 
+#if defined(MBEDTLS_SSL_CLI_C)
```

Comment:
Should we guard the definitions in `ssl_ticket.h` too?

Reply:
I have another PR that re-defines the ticket API